pyHS100/pyHS100.py

```diff
@@ -19,10 +19,16 @@
 import enum
 
 from .protocol import TPLinkSmartHomeProtocol
+from . import schedule
+from . import countdown
+from . import antitheft
 
-_LOGGER = logging.getLogger(__name__)
 
+_LOGGER = logging.getLogger(__name__)
 
+NoScheduleFound = schedule.NoScheduleFound
+NoCountdownFound = countdown.NoCountdownFound
+NoAntiTheftFound = antitheft.NoAntiTheftFound
```

Comment:
These are for the exceptions? Maybe we should have a separate exception file containing all our custom exceptions instead of splitting them into different submodules and importing them like this.